sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/Microsoft.Azure.WebJobs.Extensions.EventHubs.Tests.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.Azure.Management.Storage" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" VersionOverride="3.0.23" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" />
```

Comment:
(extreme) nit:  the slashes used in lines 30-32 are inconsistent between forward and back.